### ISSUE TITLE
Add to the Request object the ability to get a parameter from multiple possible sources

### ISSUE THREAD

**Issue by V-ed** [Vhoyon/Discord-Bot#33]
# Problem
Currently, there is only one way to get the parameter of  let's say the command `!!timer --h 1 --hours 2` ; we would need to do two calls to `getParameter()` (_with the `"h"` and `"hours"` method parameters_) which is cumbersome as both of these should actually be mapped to bring the same parameter.

We currently have duplicate parameters handling (such as `!!timer --h 1 --h 2`) but not for variant duplicate parameters.

# Solution
Basically, I'm thinking about adding a new method overloading `getParameter()` with a vararray of String that will allow to get the value of the first parameter found.

### Example of the above :
`!!timer --h 1`

`!!timer --hours 1`

These two commands should pass through the same process as they are the very same command.

I expect to be able to call `getParameter("h", "hours")` to get either one of those for the first value found.
